lib/heap/heap_info.c

```diff
@@ -66,8 +66,8 @@ static void heap_print_info(struct z_heap *h, bool dump_chunks)
 	overhead = total - free_bytes - allocated_bytes;
 	printk("\n%zd free bytes, %zd allocated bytes, overhead = %zd bytes (%zd.%zd%%)\n",
 	       free_bytes, allocated_bytes, overhead,
-	       (1000 * overhead + total/2) / total / 10,
-	       (1000 * overhead + total/2) / total % 10);
+	       (1000 * overhead + total / 2) / total / 10,
+	       (1000 * overhead + total / 2) / total % 10);
```

Comment:
This is kind of funny, because the tool is just enforcing spacing but as a result, it misses the operator precedence "violation" (i.e. the tool is inconsistent).

Reply:
True this needs some investigation.